cmake/Modules/FindCustomBlas.cmake

```diff
@@ -1,6 +1,6 @@
 # Distributed under the OSI-approved BSD 2-Clause License.
 #
-# Copyright (C) 2021 DFTB+ developers group
+# Copyright (C) 2024 DFTB+ developers group
```

Comment:
Unless the file substantially changed, the 2021 date, or possibly a range, is more appropriate.

Reply:
I have reverted the changes, including the `FindCustomLapack.cmake` case.